completers/jj_completer/cmd/util_exec.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"github.com/carapace-sh/carapace"
+	"github.com/carapace-sh/carapace-bridge/pkg/actions/bridge"
+	"github.com/spf13/cobra"
+)
+
+var util_execCmd = &cobra.Command{
+	Use:   "exec",
+	Short: "Execute an external command via jj",
+	Run:   func(cmd *cobra.Command, args []string) {},
+}
+
+func init() {
+	carapace.Gen(util_execCmd).Standalone()
+
+	util_execCmd.Flags().BoolP("help", "h", false, "Print help (see more with '--help')")
+	utilCmd.AddCommand(util_execCmd)
+
+	carapace.Gen(util_execCmd).PositionalCompletion(
+		carapace.Batch(
+			carapace.ActionExecutables(),
```

Comment:
Files as well? Executables only contains those from `PATH`.

```go
carapace.ActionExecutables(),
carapace.ActionFiles(),
```
If not, Batch isn't necessary.

Reply:
It also allows files like `./script.sh`, I've added ActionFiles but is there a way to filter ActionFiles to those with only the +x bit?